main/e2e-testing.md

```diff
@@ -143,3 +143,142 @@ Some examples projects utilizing `@agoric/synpress` for e2e tests:
 
 - [PSM dApp E2E Tests](https://github.com/Agoric/dapp-psm/tree/main/tests/e2e)
 - [Wallet dApp E2E Tests](https://github.com/frazarshad/wallet-app/tree/main/test/e2e)
+
+## @agoric/synpress Commands
+
+The `@agoric/synpress` commands are custom Cypress commands designed for use with Keplr. They provide ready-made functions to interact with Keplr-specific elements and workflows.
+
+## `setupWallet`
+
+Initializes a Keplr wallet for testing. You can either create a new one or import an existing one using the provided options.
+
+**Arguments:**
+
+(All arguments are optional)
+
+The command takes a single object as an argument with the following keys:
+
+- `secretWords` _(**string**)_: The secret words for the wallet. (default is a predefined mnemonic phrase that represents a wallet address on the Agoric chain)
+- `privateKey` _(**string**)_: The private key for the wallet.
+- `password` _(**string**)_: The password for the wallet. (default is `Test1234`).
+- `newAccount` _(**boolean**)_: Whether to create a new wallet or not. (default is `false`).
+- `walletName` _(**string**)_: The name of the wallet. (default is `My Wallet`).
+- `selectedChains` _(**array**)_: The chains to select. (default is an empty array, `[]`).
+- `createNewWallet` _(**boolean**)_: Whether to create a new wallet (default is `false`).
+
```

Comment:
`newAccount` and `createNewWallet` seem redundant and might cause ambiguity. I've created an issue [over here](https://github.com/agoric-labs/synpress/issues/47) and we will address it soon.